### DISCUSSION TITLE
Can't import Autoscale class

### DISCUSSION THREAD

**Discussion by jssche** [databricks/databricks-sdk-py#421] [Q&A]
Hi team,

I tried to import Autoscale class but kept running into an import error. There is no issue importing other classes though. I am using 0.12.0 version. 

![image](https://github.com/databricks/databricks-sdk-py/assets/55577584/855b16e0-925c-4a34-9284-7ceb2fabe6e2)

**Comment by NodeJSmith** [comment]
Looks like it's defined as `AutoScale`, capital S.
